terraform/files/bin/apply_kubeapi_cidrs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# fixup_kubeapi_cidrs.sh
+# Patch $2 (${CLUSTER_NAME}-config.yaml) to enforce API access restrictions (if enabled)
+# Reference: https://cluster-api-openstack.sigs.k8s.io/clusteropenstack/configuration.html#restrict-access-to-the-api-server
+# (c) Kurt Garloff, 03/2023
+# SPDX-License-Identifier: Apache-2.0
+
+# Test if passed list is empty
+empty_list()
+{
+	if test -z "$1" -o "$1" = "null"; then return 0; fi
+	if test "$1" = "[]" -o "$1" = "[ ]" -o "$1" = "[  ]"; then return 0; fi
+	return 1
+}
+
+get_own_fip()
+{
+	NETS=$(openstack server list --name "${PREFIX}-mgmtcluster" -f value -c Networks)
+	FIP=${NETS##*, }
```

Comment:
I found the same problem as [here](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/342) with `]}` at the end

Reply:
Good point, need to make this work with openstackclient versions >= and < 5.5.

We don't offer ovn LB in front of kube-api yet, the cloud.conf settings only affect occm created LBs (k8s objects of service type LoadBalancer).
So we can mention it, but it's not something that anyone would run into without extra manual tuning.
Still: good catch!

Reply:
> We don't offer ovn LB in front of kube-api yet, the cloud.conf settings only affect occm created LBs (k8s objects of service type LoadBalancer).

Thank you for pointing this out. CAPO has to be configured specially to use ovn lb-provider.

Reply:
Comment has been added and (more importantly) the regexp adjusted to work with both pre-5.5 and 5.5+ openstackclien network output format.t